packages/wonder-blocks-tooltip/components/tooltip.test.js

```diff
@@ -1,4 +1,397 @@
-//@flow
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Body, HeadingSmall} from "@khanacademy/wonder-blocks-typography";
+
+import Tooltip from "./tooltip.js";
+import TooltipBubble from "./tooltip-bubble.js";
+import TooltipContent from "./tooltip-content.js";
+
+const mockIDENTIFIER = "mock-identifier";
+
+jest.mock("./tooltip-bubble.js");
```

Comment:
🤷‍♂️ Does this have any impact beyond this file? Do we want to localize this mock to just being inside this test?

Reply:
It is only this file that it impacts.